aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -293,6 +321,7 @@ TuningStatus TuningResultsValidator::ValidatePyTorchVersion(const std::string& v
 TuningContext::TuningContext() :
     enable_{false},
     tuning_enable_{true},
+    record_untuned_enable_{true},
```

Comment:
```suggestion
    record_untuned_enable_{false},
```
Please have users opt-in to use this feature.